migration/20200127-remove-work-random.sql

```diff
@@ -0,0 +1,6 @@
+-- Remove indexes that haven't been used for a very long time.
+drop index if exists ix_works_audience_target_age_quality_random;
```

Comment:
These are the materialized views that were created before?

Reply:
These indexes actually predate the materialized view. They're from back when we were generating grouped feeds with direct database queries. They haven't been necessary for many years.